app/controllers/todos_controller.rb

```diff
@@ -397,7 +397,7 @@ def change_context
     @todo = current_user.todos.find(params[:id])
     @original_item_context_id = @todo.context_id
     @original_item = current_user.todos.build(@todo.attributes)  # create a (unsaved) copy of the original todo
-    @context = current_user.contexts.find(params[:todo][:context_id])
+    @context = current_user.contexts.find(params[:todo][:context_id].sub! 'c', '')
```

Comment:
why would there be a 'c' in the context id from the form?

Reply:
no clue, they were present in my vanilla install, and were preventing drag & drop from working.